install/kubernetes/addons/grafana.yaml

```diff
@@ -26,7 +26,7 @@ spec:
     spec:
       containers:
       - name: grafana
-        image: gcr.io/istio-testing/grafana:1bc30a23190aa58635d02ff7fd31bf74de0d011e
+        image: gcr.io/istio-testing/grafana:1af0c4d990ff6bb881c0243b802ed4a8d96f1a5a
```

Comment:
^^^

Reply:
This PR needs at least the given level of Mixer. Once that [PR](https://github.com/istio/mixer/pull/1256) is merged, I will remove the specific sha. 
I will not merge this PR with this sha.